package.json

```diff
@@ -39,7 +39,6 @@
     "kstreams",
     "kafka streams",
     "schema",
-    "schema registry",
```

Comment:
Unrelated to this PR; came up doing a search for "Schema Registry" label usage and can be removed since it's a lowercase version of what we already have